assets/js/stripe-payment-request.js

```diff
@@ -3,7 +3,9 @@ jQuery( function( $ ) {
 	'use strict';
 
 	var stripe = Stripe( wc_stripe_payment_request_params.stripe.key, {
-		locale: typeof wc_stripe_params !== 'undefined' ? wc_stripe_params.stripe_locale : 'auto',
+		locale: typeof wc_stripe_params !== 'undefined' ? wc_stripe_params.stripe_locale
```

Comment:
This is relying on the global params intended for a different script, and even though we do want to load that one whenever this one is loaded, there is otherwise no hard dependency between them – think we should just include the locale in with `wc_stripe_payment_request_params`?

Reply:
Yeah makes sense! Fixed in a40e6f9.